website/templates/public/login.mako

```diff
@@ -42,34 +46,36 @@
             <div class="form-group" style="padding-top: 15px">
                 <div class="text-center m-t-lg">
                     %if redirect_url:
-                        <p>For non-institutional login, click <a href="/login/?redirect_url=${redirect_url}">here</a>.</p>
+                        <p>For non-institutional sign in, click <a href="/login/?redirect_url=${redirect_url}">here</a>.</p>
                     %else:
-                        <p>For non-institutional login, click <a href="/login/">here</a>.</p>
+                        <p>For non-institutional sign in, click <a href="/login/">here</a>.</p>
                     %endif
                 </div>
             </div>
         </div>
     </div>
+%endif
+%if campaign != "institution" or not enable_institutions:
+    %if existing_user:
+        <div class="col-sm-6 col-sm-offset-3 existing-user-signin p-b-md m-b-m">
```

Comment:
Check the grid here too; the other `elif` and `else` branches have an element that takes 6 columns; this element takes 9, which is suspicious.